public/sw.js

```diff
@@ -59,19 +64,22 @@ self.addEventListener('install', (event) => {
 	}));
 });
 
-self.addEventListener('activate', (event) => {
+self.addEventListener('activate', event => {
 	event.waitUntil(
-		caches.keys().then(
-			(cacheNames) => Promise.all(cacheNames.map(
-				(cacheName) => {
-					let res;
-					if (version !== cacheName) {
-						res = caches.delete(cacheName);
-					}
-					return res;
-				},
-			)),
-		).then(self.clients.claim()),
+		caches
+			.keys()
+			.then(cacheNames =>
+				Promise.all(
+					cacheNames.map(cacheName => {
+						let res;
+						if (version !== cacheName) {
+							res = caches.delete(cacheName);
+						}
+						return res;
+					})
+				)
+			)
+			.then(self.clients.claim())
```

Comment:
Why all these changes? Can we anyhow enable eslint on this file as before?

Reply:
I want to access the localforage user object here, to check status for handling which notification to display and eslint will show error if I import it using the include method. I will revert these changes to it's previous state.